README.md

```diff
@@ -1,22 +1,53 @@
 [![codecov](https://codecov.io/gh/center-for-threat-informed-defense/attack-flow/branch/main/graph/badge.svg?token=MSGpc9mM6U)](https://codecov.io/gh/center-for-threat-informed-defense/attack-flow)
 
 # Attack Flow
-Attack Flow helps executives, SOC managers, and defenders easily understand how attackers compose adversary techniques into attacks by developing a representation of attack flows, modeling attack flows for a small corpus of incidents, and creating visualization tools to display attack flows.
-
-At the end of the project, the visualization tools will demonstrate how attack flows will help security leaders explain defensive posture to executives, help defenders understand lessons-learned in an incident, and help red-teamers easily compose realistic adversary emulation scenarios.
-
-Resources:
-* [Attack Flow Specification](/docs/attack-flow-schema.md)
-* [Attack Flow JSON Schema](/schema/attack-flow-2022-01-05-draft.json)
-* [Attack Flow Example](/schema/attack-flow-example.json)
+The Attack Flow project helps defenders move from tracking adversary behaviors individually to the sequence of techniques adversaries use to achieve their goals. Understanding the context within those sequences, as well as the relationships among them, enables additional defensive capabilities that make defenders much more effective. The project seeks to demonstrate how attack flows can explain defensive posture to executives, aid defenders understand lessons-learned from an incident, and support red-teamers easily compose realistic adversary emulation scenarios.
```

Comment:
> aid defenders understand lessons-learned

Missing a comma or a word here?

> support red-teamers easily compose

Also here.

Reply:
Should be changed to make it more readable - added a couple 'to's